tomviz/python/BinaryThreshold.py

```diff
@@ -3,8 +3,7 @@
 
```

Comment:
Below is an example of an ITK operator using the new `transform()` API. Let me know what you think...

@cjh1 @cryos